src/devices/tuya.ts

```diff
@@ -2043,7 +2042,7 @@ const definitions: DefinitionWithExtend[] = [
                         cancel: tuya.enum(2),
                     }),
                 ],
-                [3, 'soil_moisture', tuya.valueConverter.raw],
+                [3, 'humidity', tuya.valueConverter.raw],
```

Comment:
Since this is a soil moisturising sensor, `soil_moisture ` looks fine to me

Reply:
I think so too. But I've received some emails asking to change to humidity.